### ISSUE TITLE
mimic3 voices fail to download

### ISSUE THREAD

**Issue by garrett** [mkiol/dsnote#127]
I had some mimic3 voices previously downloaded in Speech Note before. However, on a fresh installation, I tried downloading a few text to speech voices and noticed that mimic3 voices are now failing to download. I guess the URLs changed?

For example, the UK apope voices:

![showing the details for the apope voices](https://github.com/mkiol/dsnote/assets/10246/37edf88e-a3cb-414a-b2c1-452098c5f240)

Trying to download, I get this:

![error](https://github.com/mkiol/dsnote/assets/10246/c6d98fc9-c068-49ed-bf8e-16a5f3798a1d)

"Error: Couldn't download the model file."

If we look at the config.json (as an example), the _latest_ version is at:

https://raw.githubusercontent.com/MycroftAI/mimic3-voices/master/voices/en_UK/apope_low/config.json

(Non-raw, GitHub UI version is at https://github.com/MycroftAI/mimic3-voices/blob/master/voices/en_UK/apope_low/config.json)

So, assuming you'd want the same commit (although it probably wouldn't hurt to bump it up to a new version), the URLs should change. Here's the working URL for the JSON (as an example):

https://raw.githubusercontent.com/MycroftAI/mimic3-voices/b239a9084e21fbaa7ac78ea6e31f5de1c31c8f42/voices/en_UK/apope_low/config.json

The fix looks like it would be to change the hostname from `github.com` to `raw.githubusercontent.com` and to also remove `raw/` from the URL (as it's included in the hostname).

I think the original URLs might still work in a browser, but that does redirects and I'm not sure if redirects are handled in the app.

**Comment by garrett** [comment]
Note: This may also affect more than just mimic3 voices, if other voices are downloaded from GitHub.

**Comment by mkiol** [comment]
Many thanks for reporting this.

The problem seems to be related only to big in size *.onnx files. It is not possible to download them. Even `git clone` on mimic3 repository doesn't work. I'm receiving the following error which actually explains everything:

```
This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
```

The problem is only with mimic3.

**Comment by mkiol** [comment]
I've created [this](https://github.com/MycroftAI/mimic3-voices/issues/12) to report the problem.

**Comment by mkiol** [comment]
I've re-uploaded all mimic3 models to my own repository on hugging face: https://huggingface.co/mukowaty/mimic3-voices

Fix: 8442c911ebe2767ac4e381aafe5fb6072b343dc2

If you want to fix this without waiting for the next release, you can edit `~/.var/app/net.mkiol.SpeechNote/data/net.mkiol/dsnote/models.json` file and update urls manually by replacing all occurrences  of:

```
https://github.com/MycroftAI/mimic3-voices/raw/b239a9084e21fbaa7ac78ea6e31f5de1c31c8f42
```

to

```
https://huggingface.co/mukowaty/mimic3-voices/resolve/2b85a29b20717fdb3da37c96c9058f1bf2dd476f
```

**Comment by mkiol** [comment]
New version 4.5.0 is out. The fix is included.
